### DISCUSSION TITLE
Can't run build-all

### DISCUSSION THREAD

**Discussion by MarcBaeuerle** [GoogleChrome/lighthouse#15975] [Q&A] [answered]
Hello,
I would like to contribute to this project, and followed the setup guide, but `yarn build-all` doesn't seem to work. I am getting a `Usage Error: Invalid destination 'lighthouse'; Can't link the project to itself` which stems from the `yarn link lighthouse` command inside `yarn reset-link`. I've made sure to update yarn and tried reinstalling it, but can't seem to fix the issue. I didn't put this under the issues tab because it seems no one else has the same problem and its a step everyone has to go through so I'm not sure where to go from here.
Thank you for your help!

**Comment by connorjclark** [comment, marked as answer]
I'm using yarn 1.22.19. Yarn v2 and v3 aren't something we ever updated to.

**Reply by MarcBaeuerle** [reply]
That fixed it, thank you!
